### ISSUE TITLE
Suggestions for the "Now playing"-window

### ISSUE THREAD

**Issue by MPconta** [abarisain/dmix#749]
When opening a whole album from an artist, the information you get in the "now playing"-window is not very satisfying. You don't find the total-time of the whole album and for the different songs the wrong information is provided. You find for every song the same cover-pic and the same album-name. Information about the track-number and track-duration would be much more helpful at this place.

Maybe it would be a good idea to introduce a choice for an "now playing"-window for a list of songs from different artists/albums and one for whole albums? 
Do you think it would be difficut to realize a switch for that in the settings, similar to the existing switch tablet-view/no tablet-view?

I know, it is a little oldfashioned to listen to whole albums, but i think there could be a lot of people using MPDroid, who would like this feature.

**Comment by abarisain** [comment]
I think you'll be interested in hurzl's branch. He maintains something like that.

**Comment by MPconta** [comment]
Thanks for the quick answer.
That sounds promising! :-)

**Comment by MPconta** [comment]
Is there already a version of Hurzl's branch to download and test?
And where is it possible to find information about this branch. It sounds very interesting to me.

And a further idea idea for the "now playing"-window:
In the settings now it is possible to choose the tags "year", "artist" and the file properties to be displayed. Would it be difficult to introduce the same possibility for the tag "coment"? I think it is widely used and a lot of users would like to see the content of it when playing a song.

**Comment by ChaoticMind** [comment]
@MPconta probably [this](https://github.com/hurzl/dmix).

**Comment by MPconta** [comment]
Hmmm...
I am not really in this github-thing, but everything on this site looks a little outdated.
Is there a possibility to install the Hurzl-MPDroid on the PlayStore?
Or how is it possible to get in contact with this project?
Thank you!

**Comment by hurzl** [comment]
So you want an apk? http://filehorst.de/d/bcDIovAd

**Comment by MPconta** [comment]
Thanks for the link Hurzl!
But sorry - it does not install on my Nexus10. Is it necessary to uninstall the actual MPDroid?
And yes, i am not really a android-genius... ;-)

**Comment by abarisain** [comment]
Yeah I think so

2015-09-12 19:21 GMT+02:00 MPconta <notifications@github.com>:

**Comment by hurzl** [comment]
Probably it thinks it goes back in version (why?) 
If you install via adb you can use the -r switch to override that:
adb install -r file.apk

**Comment by jcnoir** [comment]
I also like the idea of having the tracks that belong to the same album regrouped to have a big cover instead of several little ones.
Just for that I think it would improve the user experience. 

Is it a deliberate choice not to have that in MPDroid ? even by default ?  Or it is just that  no one did it yet ?

**Comment by abarisain** [comment]
Yup, it's a deliberate choice made by me, for a couple of reasons.
 - I think it complexifies the UI and the code
 - Makes reordering harder
 - When I took this decision, I mostly listened to tracks by different
artists with different covers.

Most softwares display their queue in the same way as we do (not that it is
a really valid point)

2015-09-13 19:45 GMT+02:00 jcnoir <notifications@github.com>:

**Comment by jcnoir** [comment]
OK Thanks for the explanation.

Maybe a nice addition in that spirit would be just to have a quick way to display the album corresponding to a track of the current playlist.
I often do that to get the the bigger image and read some details (musician names ...) but it is not very handy for now.
You have to go to the little contextual menu at the right of the track, then click on go to ..., then click on go to the album. Any idea ?
